src/library/GenerateNominations/index.tsx

```diff
@@ -194,10 +194,13 @@ export const GenerateNominations = ({
   };
 
   const disabledMaxNominations = () => {
-    return nominations.length >= maxNominations;
+    return maxNominations.isLessThanOrEqualTo(nominations.length);
```

Comment:
Why has `maxNominations` been replaced with `nominations.length`? These are 2 different values (hopefully the names are self explanatory. Please fix.

Reply:
I'm trying to understand the difference, as the results should be the same, i switched the two values positions

Reply:
The results are not the same because the = will no longer apply when you switch the values around, a >= b switched is b < a.